printer/kubectl_diff.go

```diff
@@ -0,0 +1,39 @@
+package printer
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/kubecolor/kubecolor/config"
+	"io"
+	"strings"
+)
+
+type DiffPrinter struct {
```

Comment:
You'll need to register this printer in `printer/kubectl_output_colored_printer.go` so that it'll be used

Reply:
I think we already did this see:  https://github.com/kubecolor/kubecolor/pull/196/files#diff-6fda8dfbacb16a290308a43224a4965a24238e2a09a7048b24b1d15a37c02d87R132-R133

Reply:
Ah, I missed that. Then ignore this thread :)